src/engine/framework/TestRunner.cpp

```diff
@@ -0,0 +1,26 @@
+#include <gtest/gtest.h>
```

Comment:
imo, we should just make a Test app. Similar to how we have the "DummyApp" to just run tests?